lib/users/data.js

```diff
@@ -31,4 +32,27 @@ const modelMerged = (userModel) => {
   return contentTypes([userConfig]);
 };
 
+/*
+ * Cleaner user data post form submission
+ *
+ * @param {object} body req.body post-form submission
+ *
+ * @returns {object} cleaned version of form data
+ */
+const cleanFormData = (body) => {
```

Comment:
## names
If I used the form-field names, that would mean the schema in the database would have to be written to match the form (`email--email` instead of `email`; `role--select` instead of `role`)

## bcrypt
Where should this happen?